src/lib/object-converter.ts

```diff
@@ -39,6 +39,10 @@ export async function transformOutput(output: OutputtedDeclarations): Promise<Ou
                     newEvents[event.name][keywords.wildcard].push(newEntry);
                 }
             } else if (property instanceof Property) {
+                // Check if property name is 'value' and throw an error if it is
+                if (propetyNameChanger(property.name) === 'value') {
+                    throw new Error(`Property name 'value' is reserved and cannot be used due to conflicts with the SDK. Please rename the property.`);
```

Comment:
Will this break functionality in extensions if they were already logging telemetry like this without a try catch around it?

Reply:
This is the `vscode-telemetry-extractor` repository which houses the static analyzer we run to extract all the GDPR metadata. This has no runtime affect on any code. The only side effect that may occur is Github Action CI may start to fail if you have a property with the name `value` as the static analyzer will throw and fail out.